src/components/TimeInput.tsx

```diff
@@ -6,44 +6,44 @@ const TimeInput = (props: { time: number; setTime: (time: number) => void }) =>
       <div className="flex flex-row">
         <button
           className="px-3 border-2 hover:bg-neutral-800 border-neutral-800 hover:border-neutral-700 rounded-l"
```

Comment:
Due to this button now implementing the decrement functionality, and having the disabled state, it needs "disabled" stylings. Just like the other one did.

```suggestion
          className="px-3 border-2 hover:bg-neutral-800 border-neutral-800 hover:border-neutral-700 rounded-l disabled:text-neutral-500 disabled:hover:bg-transparent disabled:hover:border-neutral-800 disabled:cursor-not-allowed"
```